src/db/.env

```diff
@@ -1,5 +1,5 @@
 #Postgres
-POSTGRES_USER=postgres
+POSTGRES_USER=posgres
```

Comment:
is this correct?

Reply:
No, thanks for highlighting. Added the t back in.

Reply:
Done.